freegeek-info.sh

```diff
@@ -144,7 +150,7 @@ batteryhealth2=$(upower -i /org/freedesktop/UPower/devices/battery_BAT1 | grep -
 if [[ -n "$batteryhealth" ]]; then
     echo -e "${BOLD}Battery Health:${RESET} $batteryhealth"
 elif [[ -n "$batteryhealth2" ]]; then
-    echo -e "${BOLD}Battery Health:${RESET} $batteryhealth2"
+    echo -e "${BOLD}Battery Health:${RESET} $batteryhealth2 (This computer is probably a Surface or ThinkPad, it has 2 batteries so list both)"
```

Comment:
```suggestion
    echo -e "${BOLD}Battery Health:${RESET} $batteryhealth2"
```
Eh, not super necessary. Also, I don't know if they list both on the sheet or not.

Reply:
Maybe we should change the code to find the overall battery health across them? I believe its important to tell volunteers what they're seeing.

Reply:
Remove the 'this computer is probably a surfcae or thinkpad' and just say '(This computer has 2 batteries, ask coordinator about listing both)'

We will call that good